src/main/java/randoop/main/GenTests.java

```diff
@@ -516,6 +522,60 @@ public boolean handle(String[] args) {
           codeWriter,
           GenInputsAbstract.regression_test_basename,
           "Regression");
+
+      HashSet<String> flakyTests = codeWriter.getFlakyTests();
+
+      if (flakyTests.size() > 0) {
+        HashMap<TypedOperation, Integer> testOccurrences = new HashMap<>();
+        HashMap<TypedOperation, Integer> flakyOccurrences = new HashMap<>();
+
+        // Tally occurrences of all methods for if-idf
+        // Each method is counted once if it appears in a sequence test.
+        tallyOperationsInSequences(testOccurrences, explorer.getRegressionSequences());
+
+        // TODO: cxing handle Error Test Sequence tallying.
+        // tallyOperationsInSequences(testOccurrences, explorer.getErrorTestSequences());
+
+        List<ExecutableSequence> regressionSequences = explorer.getRegressionSequences();
+        for (String flakyTestNum : flakyTests) {
+          int testNum = Integer.parseInt(flakyTestNum.substring(4)); // length of test
+          // Tests start at 001
+          ExecutableSequence flakyTest = regressionSequences.get(testNum - 1);
+          HashSet<TypedOperation> ops = new HashSet<>();
+          SimpleList<Statement> statements = flakyTest.sequence.statements;
+          for (int i = 0; i < statements.size(); i++) {
+            if (!statements.get(i).getOperation().isMethodCall()) continue;
+            ops.add(statements.get(i).getOperation());
+          }
+
+          for (TypedOperation to : ops) {
+            if (flakyOccurrences.containsKey(to)) {
+              flakyOccurrences.put(to, flakyOccurrences.get(to) + 1);
+            } else {
+              flakyOccurrences.put(to, 1);
+            }
+          }
+        }
+
+        Map<String, Double> methodFlakyPercentage = new HashMap<>();
+        List<Entry<String, Double>> sortedMethodsbyFlakiness;
```

Comment:
Declare a variable at its point of initialization when possible.  You can move this declaration to [line 570](https://github.com/randoop/randoop/pull/443/files#diff-dc322f142726487312b7b1299f931aebR570).  That helps to emphasize to readers that it isn't used in the `for` loop.